include/kernels/hisq_paths_force.cuh

```diff
@@ -669,8 +677,13 @@ namespace quda {
 
     };
 
-    template <typename Param> struct AllFiveAllSevenLink
-    {
+    template <typename Param> struct AllFiveAllSevenLinkOps {
+      static constexpr int cacheLen = Param::sig_positive ? 3 : 2;
```

Comment:
`cache_len` instead of `cacheLen`. We're working towards the underscore convention instead of camelCase for simpler variables, `nColor` is just a pervasive beast we're afraid to deal with thus far...

Reply:
6a727a700e87b08f30a3874877cd11e927b95f90